redis/go.mod

```diff
@@ -7,9 +7,12 @@ require (
 	github.com/philippgille/gokv/encoding v0.6.0
 	github.com/philippgille/gokv/test v0.6.0
 	github.com/philippgille/gokv/util v0.6.0
+	github.com/redis/go-redis/v9 v9.2.1
```

Comment:
With this update, the old v6 dependency shouldn't be required anymore, right? Did you run `go mod tidy`?

Reply:
I thought I did. Double-checking now...